e2e/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM cypress/base:10
+WORKDIR /app
+
+COPY package.json .
+COPY package-lock.json .
+COPY reporter-config.json .
+COPY cypress.json .
+ADD cypress cypress
```

Comment:
need to manually make these resources on as docker-compose in circle doesn't support mounted volumes.